README.md

```diff
@@ -1,9 +1,56 @@
+# Tenderly Web3 Actions
 
-[Tenderly](https://tenderly.co) Web3 Actions will run your code in response to on-chain (or even off-chain) events, usually on your smart contracts.
+<div align="center">
+    <a href="https://tenderly.co">
+        <img src="https://storage.googleapis.com/tenderly-public-assets/tenderly-logo-purple.png" alt="tenderly-node-extensions" width="100%" height="auto" style="background-color: #ffffffb2; padding: 10px 20px; margin-bottom: 20px; box-sizing: border-box; max-width:200px;" />
+    </a>
+</div>
 
-You can use Web3 Actions to create custom scenarios in order to further deepen your debugging process, create alerting patterns that are not available out-of-the-box in the Alerting section, automating testing or live production execution in the Simulator and Forks, or anything else that comes to (your) mind.
+<div align="center">
 
-The code Web3 Actions run is called a function. Function must be written in TypeScript (or JavaScript) and runs in Node 14 runtime. Specification of events that your action listens to is called a trigger.
+[![npm](https://img.shields.io/npm/v/@tenderly%2Factions.svg)](https://www.npmjs.org/package/@tenderly/actions)
+[![Twitter](https://img.shields.io/twitter/follow/TenderlyApp?style=social)](https://twitter.com/intent/follow?screen_name=TenderlyApp)
+[![Github](https://img.shields.io/github/stars/Tenderly/integration-samples?style=social)](https://github.com/Tenderly/integration-samples)
 
-To get started, see [docs](https://docs.tenderly.co/web3-actions/intro-to-web3-actions).
+</div>
+
+## Introduction
+
+[Tenderly](https://tenderly.co) Web3 Actions are programmable hooks for smart contract and chain events. They can also
+be serverless backends for your dapps. A Web3 Action is a regular TypeScript/JavaScript function that runs on Tenderly
+infrastructure.
+
+A Web3 Action comes with key-value storage for data you need to persist and secrets management for sensitive information
+such as API keys. You can think of Web3 Actions as infrastructure building a Web3 version of IFTTT/Zapier automation,
+allowing you to glue together your smart contracts, on-chain events, dapp frontends, APIs, and other services.
+
+![Web3 Actions](https://storage.googleapis.com/tenderly-public-assets/web3-actions/tenderly-web3-actions-breakdown.webp)
+
+Based on the trigger rules you define, external on-chain or off-chain events will trigger your Web3 Action and your
+TypeScript/JavaScript code will handle the event instantly.
+
+With Web3 Actions, you can build systems that let you be timely informed about non-trivial events. This way, you can
+improve your dapp in terms of UX and interactivity and enhance your project through the depth of information you can
+collect or share through notifications.
+
+> The code Web3 Actions run is called a **function**. Function must be written in TypeScript/JavaScript and runs in
+**Node 14** runtime. Specification of events that your action listens to is called a **trigger**.
```

Comment:
Actions now support Node 16 as well (depending on the runtime version). However I'd recommend not to put a specific Node version here, as it would require changes anytime we start/stop supporting a certain Node version.